server/app.js

```diff
@@ -23,6 +24,10 @@ app.use(express.static(join(__dirname, "public")));
 
 app.use("/", indexRouter);
 app.use("/ping", pingRouter);
+app.use("/creatematch", require("./routes/creatematch"));
+app.use("/joinmatch", require("./routes/joinmatch"));
+app.use("/nextmove", require("./routes/nextmove"));
+app.use("/gameinfo", require("./routes/ping"));
```

Comment:
Rather than having separate routers here, you want a single one containing all the routes related to matches. Also, you want to name the routes more conventionally: https://restfulapi.net/resource-naming/